sdk/test/opentelemetry/sdk/configurator_test.rb

```diff
@@ -197,6 +239,27 @@
           OpenTelemetry::SDK::Trace::Export::BatchSpanProcessor
         )
       end
+
+      it 'can be set by environment variable' do
+        with_env('OTEL_TRACES_EXPORTER' => 'jaeger') do
+          configurator.configure
+        end
+
+        skip 'how do we load this gem for testing?'
```

Comment:
Thoughts on this @robertlaurin ?

Reply:
I think there's a couple options here:
- We could just include the jaeger exporter as a test dependency in the gemfile.
- We could write our own double for testing
- We could use a [gem like this one](https://github.com/adammck/minitest-stub-const) and stub the constant `OpenTelemetry::Exporter::Jaeger::Collector::Exporter` and have it return a mock 

I think I would reach for requiring the gem as a test dependency and if it gave me any trouble would use the `minitest-stub-const` gem to just stub and return a mock.  I don't think this needs to be more than a shallow integration test to make sure it selects the right thing.